tests/kola/files/commonlib.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
```

Comment:
Replace this one with a symlink pointing to the commonlib.s files in shared folder.